From 16471c0e35ab4053444b05c03b5b76ebb173dc13 Mon Sep 17 00:00:00 2001
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Date: Sun, 28 Apr 2024 10:37:38 +0800
Subject: [PATCH 3/3] Restore PYTHONPATH bootstrap patch to ament_cmake_core.

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 209b90656..a53914471 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -24,7 +24,7 @@ endif
 DEB_HOST_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 
 # Needed to bootstrap since the ros_workspace package does not yet exist.
-export PYTHONPATH=@(InstallationPrefix)/lib/python3.10/site-packages:@(InstallationPrefix)/lib/python3.9/site-packages
+export PYTHONPATH=@(InstallationPrefix)/lib/python3.12/site-packages:@(InstallationPrefix)/lib/python3.11/site-packages
 
 %:
 	dh $@@ -v --buildsystem=cmake --builddirectory=.obj-$(DEB_HOST_GNU_TYPE)
-- 
2.41.0

